salt/cli/batch_async.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+'''
+Execute batch runs
+'''
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
+import tornado
+
+# Import salt libs
+import salt.client
+
+# pylint: enable=import-error,no-name-in-module,redefined-builtin
+import logging
+import fnmatch
+
```

Comment:
@cachedout do you know what this lint error means? https://jenkinsci.saltstack.com/job/pr-lint/job/PR-50546/28/warnings52Result/new/file.-1608698382/

Reply:
```gray% pylint --rcfile=../../.testing.pylintrc batch_async.py                                                                                                                                                                (git)-[pr/50546]  
Using config file /Users/mikeplace/devel/salt/.testing.pylintrc                                                        
************* Module salt.cli.batch_async                  
batch_async.py:16: [W1699(incompatible-py3-code), ] Incompatible Python 3 code found. Proposed fix:                    
 # pylint: enable=import-error,no-name-in-module,redefined-builtin                                                     
 import logging                                            
 import fnmatch                                            
+from salt.ext.six.moves import range                      
                                                           
 log = logging.getLogger(__name__)                         
                                                           

-----------------------------------                        
Your code has been rated at 9.89/10   ```